proxy/src/config.rs

```diff
@@ -236,6 +242,10 @@ impl<'a> TryFrom<&'a Strings> for Config {
             control_host_and_port: control_host_and_port?,
 
             event_buffer_capacity: event_buffer_capacity?.unwrap_or(DEFAULT_EVENT_BUFFER_CAPACITY),
+            metrics_retain_idle: Duration::from_millis(
+                metrics_retain_idle?.unwrap_or_else(|| DEFAULT_METRICS_RETAIN_IDLE)
```

Comment:
I think this doesn't need to be `unwrap_or_else`, 'cause it's a constant. it can just be `unwrap_or`.